GitUI/Translation/English.xlf

```diff
@@ -7360,7 +7360,7 @@ Check if file is accessible.</source>
         <target />
       </trans-unit>
       <trans-unit id="downloadGitForWindows.Text">
-        <source>Download msysgit</source>
+        <source>Download Git</source>
```

Comment:
We need to replace it on all xlf files, otherwise will need to get this re-translated, something I would like to avoid

Reply:
Updated. Can do the same for master, but if you never push to Transifex, there is no point